libs/auth-nest-tools/src/lib/jwt.strategy.ts

```diff
@@ -54,7 +54,11 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
         nationalId: actor.nationalId,
         scope: this.parseScopes(actor.scope),
       },
-      ip: String(request.headers['x-real-ip']) ?? request.ip,
+      ip: String(
+        request.headers['x-original-forwarded-for'] ??
+          request.headers['x-real-ip'] ??
+          request.ip,
+      ),
```

Comment:
Can you please post links proving that our infrastructure always sets/overrides these headers? I.e. that our users aren't able to spoof their IP address by just changing a header.

Reply:
I see that the X-Original-Forwarded-For is overwritten for all ingress routes in the nginx config but it is undocumented so I will not go further with this approach. Too risky.